ts/Core/Chart/StockChart.ts

```diff
@@ -760,7 +761,11 @@ addEvent(Axis, 'afterDrawCrosshair', function (
         posx = snap ? (point.plotX || 0) + left : e.chartX;
         posy = top + (opposite ? 0 : this.height);
     } else {
-        posx = opposite ? this.width + left : 0;
+        if (opposite) {
+            posx = left + width + this.offset;
+        } else {
+            posx = left + this.offset;
+        }
```

Comment:
This could be a little bit simplified:
```ts
posx = left + this.offset + (opposite ? width : 0);
```

Reply:
It looks much better when simplified, thanks!